docs/marketplace/dev/feature-integration-guides/202106.0/merchant-portal-core-cms-feature-integration.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Marketplace Merchant Portal Core feature + CMS integration
+last_updated: Mar 31, 2021
+description: This document describes how to integrate the Merchant Portal Core feature into a Spryker project.
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the Marketplace Merchant Portal Core + CMS feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Merchant Portal Core + CMS feature core.
+
+## Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+| -------------------- | ---------- | ---------|
+| Spryker Core         | dev-master | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
+| CMS      | 202009.0 | [Spryker CMS feature integration](https://documentation.spryker.com/docs/cms)
+| Marketplace Merchant Portal Core | dev-master | [Marketplace Merchant Portal Cor](/docs/marketplace/dev/feature-integration-guides/{{ page.version }}/merchant-portal-core-feature-integration.html)
+
+###  1) Create merchant restore password email templates
+
+Set up templates as follows:
+
+1. Create the following templates:
+
+**src/Pyz/Zed/MerchantUserPasswordResetMail/Presentation/Mail/merchant_restore_password.html.twig**
+
+```html
+{{ renderCmsBlockAsTwig(
+'merchant_restore_password--html',
+constant('APPLICATION_STORE'),
+mail.locale.localeName,
+{mail: mail}
+) }}
+```
+
+**src/Pyz/Zed/MerchantUserPasswordResetMail/Presentation/Mail/merchant_restore_password.text.twig**
+
+```html
+{{ renderCmsBlockAsTwig(
+'merchant_restore_password--text',
+constant('APPLICATION_STORE'),
+mail.locale.localeName,
+{mail: mail}
+) }}
+```
+
+### 2) Create CMS blocks
```

Comment:
Should be the part of `Data Import` section,  please re-arrange according to https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides